src/tsd/PipelineFactory.java

```diff
@@ -66,11 +66,11 @@
    * @throws Exception if the HttpQuery handler is unable to load 
    * serializers
    */
-  public PipelineFactory(final TSDB tsdb) {
+  public PipelineFactory(final TSDB tsdb, final RpcPluginsManager manager) {
```

Comment:
Needs an override for backwards compatibility, should just take the TSD and instantiate a new RpcPluginsManager.